### DISCUSSION TITLE
Can't checkout using SVN

### DISCUSSION THREAD

**Discussion by alervd** [community/community#108414] [Repositories] [answered]
### Select Topic Area

Bug

### Body

I believe all was working fine earlier. Today I've tried to checkout this [repository](https://github.com/vmssxe/vms_sifdm_cl) using the following URL: https://github.com/vmssxe/vms_sifdm_cl/trunk using TortoiseSVN and it says this:
Unable to connect to a repository at URL
 'https://github.com/vmssxe/vms_sifdm_cl/trunk'
'/vmssxe/vms_sifdm_cl/trunk' path not found.
I've also tried https://github.com/vmssxe/vms_sifdm_cl and got similar output. 
Is SVN broken or did something change?

**Comment by spenserblack** [comment, marked as answer]
GitHub no longer supports SVN: https://github.blog/changelog/2024-01-08-subversion-has-been-sunset/
